specifications/common/terminology.md

```diff
@@ -0,0 +1,83 @@
+# Terms and definitions {#terminology}
+
+The following terms and definitions introduce the core concepts, entities, and relationships that underpin a [=Dataspace=] and its communication protocols. Not all [=Dataspace=] entities have a concrete _technical_ materialization; some entities may exist as purely logical constructs.
+
+<dfn>Agreement</dfn>
+
+A concrete [=Policy=] associated with a specific [=Dataset=] that has been signed by both the [=Provider=] and [=Consumer=]. It is a result of a [=Contract Negotiation=] defining the [=Policy=] agreed to for a [=Dataset=].
+
+<dfn>Catalog</dfn>
+
+A collection of entries representing [=Offers=] that are advertised by a [=Provider=].
+
+<dfn>Catalog Protocol</dfn>
+
+A set of allowable [=Message Types=] that are used to request a [=Catalog=] from a [=Catalog Service=].
+
+<dfn>Catalog Service</dfn>
+
+A [=Participant Agent=] that makes a [=Catalog=] available and accessible to [=Participants=].
+
+<dfn data-lt="Data Service">Connector</dfn> (Data Service)
+
+A [=Participant Agent=] that performs [=Contract Negotiation=] and [=Transfer Process=] operations with other [=Connectors=], by implementing [=Dataspace Protocols=]. It produces [=Agreements=] and manages [=Dataset=] sharing. 
+
+<dfn>Consumer</dfn>
+
+A [=Participant=] that requests access to an offered [=Dataset=].
+
+<dfn>Contract Negotiation</dfn>
+
+A set of interactions between a [=Provider=] and [=Consumer=] that establish an [=Agreement=]. It is an instantiation of the state machine of a [=Contract Negotiation Protocol=]. An outcome of a Contract Negotiation may be the production of an [=Agreement=].
+
+<dfn>Contract Negotiation Protocol</dfn>
+
+A set of allowable [=Message Type=] sequences defined as a state machine.
+
+<dfn>Dataset</dfn>
+
+Data or a technical service that can be shared by a [=Participant=].
+
+<dfn>Dataspace</dfn>
```

Comment:
We should align this definition with the definition from the 20151 dataspaces concepts and characteristics:
'' governance framework and supporting services to build trustworthiness and enable the sharing of data through an agreed set of policies, semantic models, protocols and processes ''

Reply:
Will open a new issue for this.

Reply:
Can we discuss this in a separate issue and merge this as-is since the PR has been active for some time?

Reply:
Already opened a new issue for that.